src/app/components/Header.js

```diff
@@ -14,6 +14,7 @@ import {
 
 const HeaderBar = styled.div`
   background-color: var(--brandBackground);
+  border-left: solid 2px var(--grayBorderMain);
```

Comment:
This was added temporarily, to keep the item page looking the same until we get around to those layout changes